packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.controller.js

```diff
@@ -0,0 +1,84 @@
+export default class VpsScheduleBackupCtrl {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, Poller, VpsService) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.Poller = Poller;
+    this.VpsService = VpsService;
+    this.model = {
+      rescheduleTimepicker: null,
+    };
+  }
+
+  $onInit() {
+    this.model.rescheduleTimepicker = this.automatedBackupOption.schedule;
+    this.isLoading = false;
+  }
+
+  /**
+   * Wait the triggered task
+   * @param taskId {Number}: id of running task
+   * @returns {Promise}: task promise
+   */
+  waitRescheduleAutomatedBackupTask(taskId) {
+    const endPointUrl = `/vps/${this.serviceName}/tasks/${taskId}`;
+    return this.Poller.poll(endPointUrl, null, {
```

Comment:
I don't see the Poller getting killed when controller is destroyed

Reply:
Completed.